### ISSUE TITLE
Memoria.FF6PR: Output log created, no useable features present in game

### ISSUE THREAD

**Issue by EMULLC** [Albeoris/Memoria.FFPR#43]
[LogOutput.log](https://github.com/Albeoris/Memoria.FFPR/files/9305568/LogOutput.log)

Everything worked well utilizing the ff5pr until yesterday, and today, cannot get memoria features to be useable in game now on FF6PR. Any help appreciated.

Step by step starting with installing the custom unity, then bes, then memoria combining the folders. no dice.

**Comment by Albeoris** [comment]
Hi. **Remove BeepInEx** folder first, and install all again.

**Comment by EMULLC** [comment]
attempted to remove the bepin folder first then install again, no changes unfortunately. 

Have also tried removing the zip files and redownloading, then installing bep then memoria replacing files, no luck. At a complete loss

**Comment by Albeoris** [comment]
Checked on a clean install of FF5/6, BepInEx, Memoria - everything works fine.
Your logs also do not contain errors.

Do you use the latest version?
https://github.com/Albeoris/Memoria.FFPR/releases

**Comment by mrgrim** [comment]
Sorry to necro this, but I'm having the exact same problem and am wondering if you ever worked it out. Log shows patches load fine, Memoria receives events like OnDestroy(), but none of the patches actually execute. Also just like you, one day it was working, the next it wasn't.

The only two suspicious events are a windows update was installed and a game update was released. I uninstalled the windows update, and I replaced the game update with a backup of 1.0.6. Out of desperation I tried updating BepinEx to 6.0.0-pre1. Log attached.

[LogOutput.log](https://github.com/Albeoris/Memoria.FFPR/files/14353578/LogOutput.log)
